src/UKHO.S100PermitService.Common/Encryption/S100Crypt.cs

```diff
@@ -39,7 +41,7 @@ public IEnumerable<ProductKey> GetDecryptedKeysFromProductKeys(IEnumerable<Produ
                 });
             }
 
-            _logger.LogInformation(EventIds.GetDecryptedKeysFromProductKeysCompleted.ToEventId(), "Get decrypted keys from product keys completed.");
+            _logger.LogInformation(EventIds.GetDecryptedKeysFromProductKeysCompleted.ToEventId(), "Get decrypted keys from Product keys completed.");
```

Comment:
To keep it consistent as line no. 30 no need to change
```suggestion
            _logger.LogInformation(EventIds.GetDecryptedKeysFromProductKeysCompleted.ToEventId(), "Get decrypted keys from product keys completed.");
```

Reply:
done